js/components/main.js

```diff
@@ -532,6 +539,10 @@ class Main extends ImmutableComponent {
     windowActions.setClearBrowsingDataDetail()
   }
 
+  onHideImportBrowserDataPanel () {
+    windowActions.setImportBrowserDataDetail()
```

Comment:
along the same lines as the previous comment this would be something like `windowActions.importDataPanelHidden`